packages/database-mongo-sessions/src/mongo-sessions.ts

```diff
@@ -61,6 +69,23 @@ export class MongoSessions implements DatabaseInterfaceSessions {
       timestamps: { ...defaultOptions.timestamps, ...options.timestamps },
     };
 
+    if (typeof this.options.idProvider === 'function') {
```

Comment:
The issue I see of this check is that currently this option is inherited from the @accounts/mongo package, see https://github.com/accounts-js/accounts/blob/master/packages/database-mongo/src/mongo.ts#L58. The user might use both `idProvider` and `idSessionProvider` and this condition would be triggered.
We should probably remove this totally.

Reply:
I have removed this and fixed a failing test to use `idSessionProvider` since it was previously using `idProvider`. This is addressed in b89b6cdf77c759f9dc2dc7de936becf805184259.